pygluu/kubernetes/templates/helm/gluu/charts/fido2/templates/deployment.yml

```diff
@@ -73,6 +73,8 @@ spec:
           runAsNonRoot: true
         {{- end }}
         env:
+        - name: GLUU_JAVA_OPTIONS
+          value: "-XX:MaxDirectMemorySize={{ (.Values.resources.limits.memory | replace "Mi" "") }}m -Xmx{{sub (.Values.resources.limits.memory  | replace "Mi" "") 300}}m"
```

Comment:
I think we should introduce `gluuCustomJavaOptions` to all java-based components, pre-calculate the values in helpers, and load it in template

Reply:
Yes because we do have people using the `GLUU_JAVA_OPTIONS` override env in the values.yaml. This would cause a clash.